.github/ISSUE_TEMPLATE/add_conference.yml

```diff
@@ -11,27 +11,39 @@ body:
       placeholder: <CONFERENCE NAME>
     validations:
       required: true
+  - type: textarea
+    id: description
+    attributes:
+      label: Summary
+      description: Provide a summary (2-5 sentences) of the announcement information
+      placeholder: The <CONFERENCE> conference is happening in <LOCATION> on <DATE>. The CFP is open until <DATE>.
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: |
+        ## Other Information 
+        Only fill out the information below that is applicable to your announcement.
   - type: input
     attributes:
-      label: URL
+      label: Conference Website
       description: The URL of the conference.
       placeholder: https://example.com
     validations:
-      required: true
+      required: false
   - type: input
     id: conference_dates
     attributes:
       label: Conference Dates
       description: The start and end date of the conference.
-      placeholder: "DD (MMM) -DD MMM YYYY"
-  - type: dropdown
-    id: conference_type
+      placeholder: "DD (MMM)-DD MMM YYYY"
+  - type: checkboxes
+    id: online-option
     attributes:
-      label: Conference Type
-      description: Is the conference in-person online or both?
-      options: ['both', 'in-person', 'online']
-    validations:
-      required: true
+      label: Online Tickets
```

Comment:
I'd recommend changing this from "Tickets" to something like "Participation" or "Attendance" for clarity.

Reply:
Solution to this depends on the first response if we remove the bottom portion of this then this goes away.